### PR TITLE
[15.0][MIG] website_sale_coupon_page: Migration to version 15.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
cc@Tecnativa  TT39019

@sergio-teruel @pedrobaeza @chienandalu   please review!

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
Very good job @stefan-tecnativa :smile: :+1: ! Once you fix pre-commit, it'll be ok

File: website_sale_coupon_page/templates/promotion_templates.xml [hunk 1]
```diff
@@ -37,13 +37,13 @@
...
                         class="card-img-top img-fluid"
-                        t-attf-src="/web/image/sale.coupon.program/#{promo['id']}/image_1920"
+                        t-attf-src="/web/image/coupon.program/#{promo['id']}/image_1920"
                         t-att-alt="'%s' % promo['id']"
                         t-if="promo['image_1920']"
                     />
                 </a>
                 <div class="card-body" t-if="promo['public_name']">
-                    <div class="card-text" t-raw="promo['public_name']" />
+                    <div class="card-text" t-out="promo['public_name']" />
```
Comment (pedrobaeza) [inline website_sale_coupon_page/templates/promotion_templates.xml:46]: Is this putting the HTML content correctly?

File: website_sale_coupon_page/tests/test_website_sale_coupon_page.py [hunk 2]
```diff
@@ -6,8 +6,10 @@
 from PIL import Image
 
 from odoo.tests.common import HttpCase
+from odoo.tests import tagged
 
 
+@tagged('post_install', '-at_install')
```
Comment (pedrobaeza) [inline website_sale_coupon_page/tests/test_website_sale_coupon_page.py]: Why it should be post-install?

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Apart from the pre-commit thing.

File: website_sale_coupon_page/templates/promotion_templates.xml (hunk 1 above)
Comment (chienandalu) [reply website_sale_coupon_page/templates/promotion_templates.xml:46]: Yes:

image

File: website_sale_coupon_page/tests/test_website_sale_coupon_page.py (hunk 2 above)
Comment (pedrobaeza) [reply website_sale_coupon_page/tests/test_website_sale_coupon_page.py]: And what about this?

Comment (stefan-tecnativa) [reply website_sale_coupon_page/tests/test_website_sale_coupon_page.py]: This was included to avoid a warning during the tests.

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Oh, wait, pre-commit is still incorrect because there are changes. Run pre-commit locally, git amend, and push again.

**Pull request merged**